src/main/java/org/moltimate/moltimatebackend/util/FileUtils.java

```diff
@@ -0,0 +1,52 @@
+package org.moltimate.moltimatebackend.util;
+
+import org.biojava.nbio.structure.io.MMCIFFileReader;
+import org.biojava.nbio.structure.io.PDBFileReader;
+import org.moltimate.moltimatebackend.model.Motif;
+import org.moltimate.moltimatebackend.model.Residue;
+import org.springframework.core.io.ByteArrayResource;
+import org.springframework.core.io.Resource;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.util.List;
+
+public class FileUtils {
+
+    private static final PDBFileReader PDB_FILE_READER = new PDBFileReader();
+    private static final MMCIFFileReader MMCIF_FILE_READER = new MMCIFFileReader();
+
+    public static Motif getMotifFromFile(MultipartFile file) {
+        return null;
+
+        // TODO: Read motif metadata from file and process it
+
+//        InputStream fileInputStream;
+//        try {
```

Comment:
Changing this later?

Reply:
Thanks! I removed the commented code and left the TODO

Reply:
👍